zeroize/tests/zeroize.rs

```diff
@@ -208,3 +208,60 @@ fn asref() {
     let _asmut: &mut [u8] = buffer.as_mut();
     let _asref: &[u8] = buffer.as_ref();
 }
+
+#[cfg(not(miri))]
+#[cfg(feature = "test-allocator")]
+mod zeroization_with_custom_allocator {
```

Comment:
It maybe be worth to move this module into a separate test file, so the custom allocator would not affect the other tests.

Reply:
Sadly the `#[global_allocator]`, once enabled, does affect all tests. Until the `allocator_api` lands I know of no way around this. :/

Happy to move it to separate file though. :)

Reply:
You can use a `cfg` attribute rather than a feature. Or, use a crate like `version_check` to detect nightly

Reply:
@tarcieri If using the nightly unstable API is acceptable (and preferable) that'd make it a bit less ugly. Is that your preference? 

I did not dig too deep in that particular rabbit hole but I think that the feature is being rewritten and is not close to stabilization.

Reply:
My main concern is not introducing features for non-user-facing things like tests